test/playwright/test_ux.py

```diff
@@ -193,7 +193,7 @@ def _existing_environment_interactions(page, env_name, time_to_build_env=3*60*10
     # change the description
     page.get_by_placeholder("Enter here the description of your environment").fill("new description")
     # change the vesion spec of an existing package
-    page.get_by_role("row", name="ipykernel", exact=False).get_by_role("button").first.click()
+    page.get_by_role("row", name="ipykernel", exact=False).get_by_role("combobox").first.click()
```

Comment:
As mentioned in the other comment, the generated code now uses `combobox` instead of `button`.

Reply:
When I was reviewing this PR, I noticed that after running `yarn start` if I loaded the web app from localhost:8000, I got a combobox, whereas if I loaded the web app from localhost:8080 I got a button.

Reply:
Okay, I think I see why this happens. It looks like the [Conda Store server comes bundled with its own version of Conda Store UI](https://github.com/conda-incubator/conda-store/blob/d0dba0f52c490e98dc8f92519fd15ac1709b415f/conda-store-server/hatch_build.py#L11). Wild.

So then this change makes sense; however, I'm now concerned about a possible discrepancy between the environment variables in the GitHub runner environment versus local dev, resulting in this test passing on CI but failing locally.

Reply:
We might need to redefine the default value for `CONDA_STORE_SERVER_PORT` at the top of the file.

Reply:
Wait, when was this changed to a combobox? - I thought this was during the #310 PR

In that case both `8080` and `8000` should be returning a combobox.